### PR TITLE
Release 0.17.0

### PR DESCRIPTION
**Pull request by 2opremio**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,87 @@
...
+- Tabular view.
+	[#1673](https://github.com/weaveworks/scope/pull/1673)
+	[#1747](https://github.com/weaveworks/scope/pull/1747)	
+	[#1753](https://github.com/weaveworks/scope/pull/1753)
+	[#1774](https://github.com/weaveworks/scope/pull/1774)
+	[#1775](https://github.com/weaveworks/scope/pull/1775)
+	[#1784](https://github.com/weaveworks/scope/pull/1784)
+- Loading indicator
+	[#1485](https://github.com/weaveworks/scope/pull/1485)
+- Dont show weavescope logo when running in a frame
```
Comment (paulbellamy) [inline CHANGELOG.md]: Don't

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,95 @@
+## Release 0.17.0
+
+Highlights:
+- New Grid view as an alternative to Scopes classic graph view.  It provides
```
Comment (jml) [inline CHANGELOG.md]: s/Scopes/Scope's/

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,95 @@
...
+  higher information density, proving particularly useful when there's a high
+  node count in the graph view.
```
Comment (jml) [inline CHANGELOG.md]: "there's a high node count" is pretty inside baseball. "there are many nodes" might be more friendly.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,95 @@
...
+- Considerable Performance enhancements: the CPU efficiency of the Scope App has
```
Comment (jml) [inline CHANGELOG.md]: "Performance" should be lower case.

File: CHANGELOG.md (hunk 2 above)
Comment (rade) [inline CHANGELOG.md]: It's not a Grid, and it's not a View. "Table Mode".

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,95 @@
...
+  increased in more than 50% and the Scope probes over 25%.
+
+
+New features and enhancements:
+- Grid view.
```
Comment (rade) [inline CHANGELOG.md]: Table mode.

**Comment by 2opremio** [comment]
All good from me. I tested on: 

* The ECS guide: Things are looking good and CPU consumption is ~10% for the app and 6% for the probes (less than 25% of the numbers observed in https://github.com/weaveworks/scope/issues/897). I will let it run for a bit but I think that means we can close https://github.com/weaveworks/scope/issues/897 and https://github.com/weaveworks/guides/issues/181

* Connecting a probe to Weave Cloud: Things are rendered as expected and I don't see any backwards compatibility problems.

**Comment by idcrosby** [comment]
Testing locally, using single docker host with Socks demo app running. Looks like there are some missing connections.
screen shot 2016-08-15 at 17 54 38
[report (2).json.zip](https://github.com/weaveworks/scope/files/418414/report.2.json.zip)

**Comment by 2opremio** [comment]
@idcrosby Can you be a bit more specific? What connections were you expecting to see and aren't there?

**Comment by 2opremio** [comment]
@idcrosby Also, could you see those connections with prior releases?

**Comment by idcrosby** [comment]
Specifically between catalogue and the front-end.  I will re-test with previous version of scope

**Comment by 2opremio** [comment]
Thanks!

**Comment by idcrosby** [comment]
Ok seeing same behaviour with version 0.16.2 so this is potentially not a scope issue, and at least not a regression.

**Comment by 2opremio** [comment]
@idcrosby Thanks again. Without any urgency and when you have the time, could you please verify whether it's a bug and create a ticket if it is?

**Comment by davkal** [comment]
LGTM. new table view mode works great!

**Comment by 2opremio** [comment]
> I will let it run for a bit but I think that means we can close #897 and weaveworks/guides#181

Nah, unfortunately I still get the same problem after a few hours :S

**Comment by rade** [comment]
> Nah, unfortunately I still get the same problem after a few hours :S

Surely that makes weaveworks/guides#181 the likely culprit. Or were you running on non-micro instances?

**Comment by 2opremio** [comment]
> Surely that makes weaveworks/guides#181 the likely culprit. Or were you running on non-micro instances?

I was running micro instances, but the CPU load was pretty low (or so I thought). I'm surprised they surpassed the CPU credit limit.

**Comment by rade** [comment]
According to [The Internet](https://forums.aws.amazon.com/message.jspa?messageID=254836) one call tell whether throttling is taking place by observing the "steal" (st) metric in 'top'.

**Comment by paulbellamy** [comment]
After this release, please merge: https://github.com/weaveworks/scope/pull/1682

**Comment by 2opremio** [comment]
You can merge it already if you want, changes to the release should go in this branch

**Comment by foot** [comment]
LGTM

**Pull request merged**